streams/src/test/java/org/apache/kafka/streams/kstream/internals/InternalStreamsBuilderTest.java

```diff
@@ -156,6 +156,30 @@ public void shouldStillMaterializeSourceKTableIfMaterializedIsntQueryable() thro
         assertEquals("topic2", topology.storeToChangelogTopic().get(storeName));
         assertNull(table1.queryableStoreName());
     }
+    
+    @Test
+    public void shouldBuildGlobalTableWithNonQueryableStoreName() throws Exception {
```

Comment:
So we need `throws Exception` here? (same below)

Reply:
I am not too sure about that because the other tests all used {{throw Exception}}.